frontend/src/views/SQR/Form.vue

```diff
@@ -381,8 +410,11 @@ export default class Form extends Vue {
   }
 
   async mounted() {
-    this.canSubmit = true;
-    this.getSavedData();
+    await this.checkIsCoordinator();
+    await this.canSubmitForm();
```

Comment:
Usually the methods should be named in the form “Do this and that”. `canSubmitForm` looks more like a calculated property. Consider renaming or refactoring this.